### ISSUE TITLE
Multi sub-company support.

### ISSUE THREAD

**Issue by havietnam** [hexya-erp/hexya#9]
Does Hexya support multi sub-company?
As I understand, allocate permissions  for users by using groups is not enough (when create more and more sub-company).
Can you tell me how to? Or do you have any plan for?
Thanks in advanced.

**Comment by npiganeau** [comment]
To support multi sub-company as in Odoo, we need 3 different mechanism:

- Record Rules to automatically filter records according to some rules (in this context, according to the company) in an independent way from ACL and groups.
- Property fields to be able to have the same field have different values per company (e.g. sale tax for a product)
- Specific modules to master inter-company workflows (e.g. creating a purchase order to another company should create a sale order in that company, same for invoices, etc.)

As for now:
- Record rules mechanism are implemented
- We have plans to have property fields, but we are still thinking about how to make it an efficient way (not like Odoo where all of them are in a single table)
- We will certainly make the inter-company workflow modules too.

**Comment by hzmnet** [comment]
yes, how to make data of them alone but relate to each other is a big work.
